<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fondo1</class>
 <widget class="QMainWindow" name="Fondo1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>986</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Fondo1{
	background-color: #fff;
}
#fondoTitulo{
	background-color:#03A9F4;
	box-shadow: 10px 5px 5px black;
}
#fondoTitulo_2{
	background-color:#03A9F4;
	box-shadow: 10px 5px 5px black;
}

#titulo{
	font-size: 25px;
    color:#fff;
	font-family: Arial, Verdana;
	font-weight: bold;
}

#btnGuardar{ background: #2E7D32; color: #fff; font-size: 12px; font-weight: bold; }
#btnEliminar{ background: #B71C1C; color: #fff; font-size: 12px; font-weight: bold; }
#btnLimpiar{ background: #0277BD; color: #fff; font-size: 12px;font-weight: bold;  }
#btnBuscarImg{ background: #0277BD; color: #fff; font-size: 12px; font-weight: bold; }

#btnBuscar{ background: #2E7D32; color: #fff; font-size: 12px; font-weight: bold; }

#verImagen{
	background-color:#FAFAFA;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>431</width>
      <height>471</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblId">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtID">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblCedula">
       <property name="text">
        <string>Cédula</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtCedula">
       <property name="maxLength">
        <number>11</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblNombres">
       <property name="text">
        <string>Nombres</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtNombres"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblApellidos">
       <property name="text">
        <string>Apellidos</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtApellidos"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblFecha">
       <property name="text">
        <string>Fecha de Nacimiento</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="txtFecha"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblTelefono">
       <property name="text">
        <string>Edad</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="txtEdad">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblCiudad">
       <property name="text">
        <string>N° Aportaciones</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="txtAport">
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblNivel">
       <property name="text">
        <string>Dirección 1</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="txtDireccion1"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblNivel_2">
       <property name="text">
        <string>Dirección 2</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="txtDireccion2"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lblNivel_3">
       <property name="text">
        <string>Teléfono 1</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="txtTelefono1">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lblNivel_4">
       <property name="text">
        <string>Teléfono 2</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="txtTelefono2">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lblId_2">
       <property name="text">
        <string>E-mail</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="txtCorreo"/>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="lblCedula_2">
       <property name="text">
        <string>Sueldo</string>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QDoubleSpinBox" name="txtSueldo">
       <property name="maximum">
        <double>9999.989999999999782</double>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="lblNombres_2">
       <property name="text">
        <string>Días laborales</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QSpinBox" name="txtDias">
       <property name="maximum">
        <number>365</number>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="lblApellidos_2">
       <property name="text">
        <string>Sexo</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QComboBox" name="cbxSexo">
       <item>
        <property name="text">
         <string>Masculino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Femenino</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="lblFecha_2">
       <property name="text">
        <string>Nivel académico</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QComboBox" name="cbxNivel">
       <item>
        <property name="text">
         <string>Básica Primaria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Básica Secundaria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Formación Tecnológica</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Formación Universitaria</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Profesional</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Postgrado o Especialización</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magister, PHD</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="lblTelefono_2">
       <property name="text">
        <string>Cuenta bancaria</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLineEdit" name="txtCuenta"/>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="lblCiudad_2">
       <property name="text">
        <string>Tipo discapacidad</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QComboBox" name="cbxDiscapacidad">
       <item>
        <property name="text">
         <string>Ninguna</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Discapacidad física o motora</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Discapacidad auditiva</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Discapacidad visual</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Discapacidad intelectual o mental</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="verImagen">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>130</y>
      <width>231</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBuscarImg">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>370</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Buscar Imagen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLimpiar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="lblNivel_13">
       <property name="text">
        <string>Nombre recomendado</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtNombreRecom"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblNivel_14">
       <property name="text">
        <string>Teléfono recomendado</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtTelefonoRecom">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblNivel_15">
       <property name="text">
        <string>Celular recomendado</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtCelularRecom">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblNivel_16">
       <property name="text">
        <string>Ciudad </string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtCiudad"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnRegresar">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>420</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
